modules/serverless-eventing/pages/service-mesh/eventing-service-mesh-mt-channel-based-broker-authorization.adoc

```diff
@@ -0,0 +1,328 @@
+      ports:
+        - containerPort: 8080
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> Injects {SMProductShortName} sidecars into the pod.
+
+. Apply the consumer service resources:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+.Securing access to the Knative Broker
+
+By default, every workload is allowed to send events to a Knative Broker, with {SMProductName}, we can
+apply a policy to deny posting events by default.
+
+.Apply a deny by default authorization policy
+
+. Create a deny by default `AuthorizationPolicy` in the `knative-eventing` namespace:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: deny-all-by-default
+  namespace: knative-eventing
+spec: { } <1>
+----
+<1> Disallow any operations to every workload that is part of the service mesh in the `knative-eventing` namespace.
+
+. Apply the `AuthorizationPolicy` resource:
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Verify access is denied
+
++
+we have denied access to every workload to the knative-eventing namespace, which disallows the
+`ContainerSource` `heartbeat-source-mt-channel-based-broker` to send events to the Knative `Broker`
+`channel-based-broker-br`, therefore, we should see the following lines in the `heartbeats` pods:
+
++
+[source,terminal]
+----
+$ oc logs $(oc get pod -n authz-tests -o name | grep heartbeat-source-mt-channel-based-broker) -c heartbeats -n authz-tests
+----
++
+.Example output
+[source,terminal]
+----
+2023/06/13 10:17:04 sending cloudevent to http://broker-ingress.knative-eventing.svc.cluster.local/authz-tests/channel-based-broker-br
+2023/06/13 10:17:04 failed to send cloudevent: 403:
+2023/06/13 10:17:05 sending cloudevent to http://broker-ingress.knative-eventing.svc.cluster.local/authz-tests/channel-based-broker-br
+2023/06/13 10:17:05 failed to send cloudevent: 403:
+----
+
+.Authorize source to post events to Knative Broker
+
+In the previous section, we denied access to Knative Eventing workloads, to grant permissions to
+post events to a Knative Broker with class MTChannelBasedBroker.
+
+. Create a `AuthorizationPolicy` in the `knative-eventing` namespace to allow pods
+in the `authz-tests` namespace to send events to Knative Brokers in the same `authz-tests` namespace:
++
+[source,yaml]
+----
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-authz-tests-mt-channel-based-broker
+  namespace: knative-eventing
+spec:
+  action: ALLOW
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: "broker-ingress" <2>
+  rules:
+    - from: <1>
+        - source:
+            namespaces: [ "authz-tests" ]
+      to: <2>
+        - operation:
+            methods: [ "POST" ]
+            paths: [ "/authz-tests/*" ] <3>
+----
+<1> Allow workloads in the `authz-tests` namespace
+<2> To post events to Knative brokers in the `authz-tests` namespace.
```

Comment:
should we say that this is only true for the MT-Channel-based broker? (instead of generic Knative Brokers):